web/src/views/SubmissionPortal/Components/StudyForm.vue

```diff
@@ -48,6 +48,8 @@ export default defineComponent({
       },
     ]);
 
+    const doiProviderValues = Object.keys(NmdcSchema.enums.DoiProviderEnum.permissible_values);
```

Comment:
> 3. Updating the values displayed for each enum. I'm not sure what the best way to do that is since they're imported from the schema.

The permissible value definitions (i.e. the _values_ of the `permissible_values` object) has a `title` field that should be used for end-user display. Combine that with the ability of the `items` [prop](https://v2.vuetifyjs.com/en/api/v-select/#props-items) on `<v-select>` to take an array of objects instead of an array of strings. So something like this:

```suggestion
    const doiProviderValues = Object.values(NmdcSchema.enums.DoiProviderEnum.permissible_values)
      .map((pv: any) => ({
        value: pv.text,
        text: pv.title || pv.text, // The permissible value definition should have a title, but in case it doesn't fallback to text (which it will always have)
      }))
      .sort((a, b) => a.text.localeCompare(b.text));
```